src/set_theory/game/pgame.lean

```diff
@@ -680,6 +680,109 @@ def relabel_relabelling {x : pgame} {xl' xr'} (el : x.left_moves ≃ xl') (er :
   relabelling x (relabel el er) :=
 relabelling.mk el er (λ i, by simp) (λ j, by simp)
 
+/-- `is_relabelled_option x y` means that `x` is either a either a left or right option of `y`, up
+to relabelling. Contrast with `is_option`. -/
```

Comment:
```suggestion
to relabelling. Contrast with `pgame.is_option`. -/
```
Doc-gen will only link fully-qualified names. It's fine for a lemma to refer to itself with an incomplete name, but any kind of cross-reference should be fully-qualified.